retrofit/src/test/java/retrofit/RestAdapterTest.java

```diff
@@ -129,4 +144,60 @@
           + "Checked: [Built-in CallAdapterFactory]");
     }
   }
+
+  @Test public void missingConverterThrowsOnNonRequestBody() throws IOException {
+    RestAdapter ra = new RestAdapter.Builder()
+        .endpoint(server.getUrl("/").toString())
+        .build();
+    CallMethod example = ra.create(CallMethod.class);
+    try {
+      example.disallowed("Hi!");
```

Comment:
This test and the one below now fail on service method invocation rather than call execution with the converter validation changes :+1: